modules/web-console/web-agent/src/main/java/org/apache/ignite/console/agent/AgentLauncher.java

```diff
@@ -190,7 +190,7 @@ public static void main(String[] args) {
             websocket.awaitClose();
         }
         catch (Throwable e) {
-            log.error("Web Console Agent failed to start", e);
+            log.error("Web Console Agent failed to startWatchTask", e);
```

Comment:
Lets write this message in English: "Web Console Agent failed to start watch cluster topology task" ?